common/changes/office-ui-fabric-react/iesliderfix_2018-11-22-01-10.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Number.parseFloat() method is not supported in IE11. The fix was to use parseFloat() method instead.",
```

Comment:
Consider adding the component name prefix to this comment so that folks reading the release notes can figure out which component was modified at a glance.

>Slider: Fix Number.parseFloat() method is not supported in IE11.